extractor/src/main/java/org/schabi/newpipe/extractor/search/SearchExtractor.java

```diff
@@ -37,4 +37,15 @@ public SearchQueryHandler getLinkHandler() {
     public String getName() {
         return getLinkHandler().getSearchString();
     }
+
+    /**
+     * When you search on some service, it can give you another and corrected request.
+     * This method says if it's the case.
+     * <p>
+     * Example: on YouTube, if you search for "pewdeipie",
+     * it will give you results for "pewdiepie", then isCorrectedSearch should return true.
+     *
```

Comment:
Where is there no closing tag?

Reply:
HTML () doesn't require to close p tag. The next one closes it.
It's the same thing for Javadoc. Look at [Javadoc guidelines, getImage](https://www.oracle.com/technetwork/java/javase/documentation/index-137868.html), they didn't put a closing p tag. It's closed by `@return`, as mentionned in the guidelines: 
> + The first line that begins with an "@" character ends the description. There is only one description block per doc comment; you cannot continue the description following block tags.

Plus if you watch java source code, for example [java.lang.String](http://www.docjar.com/html/api/java/lang/String.java.html), you can see there are lots of paragraph but no closing tag.

Finally, I didn't even add the p tag, intellij did it with autoformatting, and see this [SO about p tag in javadoc](https://stackoverflow.com/a/5263598/12680950).

Reply:
I didn't know that about Javadoc. Interesting…